### PR TITLE
GracefulNodeShutdownBasedOnPodPriority is Beta in 1.24

### PR DESCRIPTION
**Pull request by tengqm**
Upstream: kubernetes/kubernetes#107986
KEP: https://github.com/kubernetes/enhancements/pull/3194

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
This isn't _wrong_; however, I imagine that a change to https://kubernetes.io/docs/concepts/architecture/nodes/ is also appropriate?

**Comment by sftim** [comment]
This is a fixup to https://github.com/kubernetes/website/pull/32771

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -534,10 +534,17 @@ next priority class value range.
...
+Using this feature requires enabling the `GracefulNodeShutdownBasedOnPodPriority`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+, and setting `ShutdownGracePeriodByPodPriority` in the
+[kubelet config](/docs/reference/config-api/kubelet-config.v1beta1/)
+to the desired configuration containing the pod priority class values and
+their respective shutdown periods.
+
+{{< note >}}
+The `GracefulNodeShutdownBasedOnPodPriority` was introduced as Alpha API in 1.24.
+In v1.24, the feature has been promoted to Beta stage and enabled by default.
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md:546]:
```suggestion
The ability to take Pod priority into account during graceful node shutdown was introduced
as an alpha feature in Kubernetes v1.23; in Kubernetes {{< skew currentVersion >}}
that feature is beta and is enabled by default.
```

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks.

I spotted a factual error about which Kubernetes release introduced `GracefulNodeShutdownBasedOnPodPriority`; I also suggested new wording.

**Comment by reylejano** [comment]
A few non-blocking nits
/approve

File: content/en/docs/concepts/architecture/nodes.md [hunk 2]
```diff
@@ -534,10 +534,18 @@ next priority class value range.
 If this feature is enabled and no configuration is provided, then no ordering
 action will be taken.
 
-Using this feature, requires enabling the
-`GracefulNodeShutdownBasedOnPodPriority` feature gate, and setting the kubelet
-config's `ShutdownGracePeriodByPodPriority` to the desired configuration
-containing the pod priority class values and their respective shutdown periods.
+Using this feature requires enabling the `GracefulNodeShutdownBasedOnPodPriority`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+, and setting `ShutdownGracePeriodByPodPriority` in the
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md:539]: nit: the comma should follow the hyperlink with no preceding whitespace

**Review by sftim — APPROVED** [review]
/lgtm

Ship it!

**Pull request merged**